Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -554,22 +529,19 @@ - (NSUInteger)batchCount {
 }
 
 /**
- * Removes entries from from the given @a batches and returns them.
+ * Removes the first n entries from the from the given batches and returns them.
  *
- * @param holes An array of indexes in the batches array; in increasing order. Indexes are relative
- *     to the original state of the batches array, not any intermediate state that might occur.
+ * @param n The number of batches to remove..
  * @param batches The array to mutate, removing entries from it.
  * @return A new array containing all the entries that were removed from @a batches.
  */
-- (NSArray<FSTMutationBatch *> *)makeHoles:(NSArray<NSNumber *> *)holes
-                                 inBatches:(NSMutableArray<FSTMutationBatch *> *)batches {
+- (NSArray<FSTMutationBatch *> *)removeFirstBatches:(int)n
+                                          inBatches:(NSMutableArray<FSTMutationBatch *> *)batches {
   NSMutableArray<FSTMutationBatch *> *removed = [NSMutableArray array];
```

Comment:
Very optional: looking at the docs, it seems like this could be expressed more clearly as `subarrayWithRange` and `removeObjectsInRange`, except creating `NSRange`s is so ugly that it won't bring a significant decrease in verbosity.